BlogTemplate/Pages/Edit.cshtml.cs

```diff
@@ -55,6 +58,10 @@ public IActionResult OnPostPublish([FromRoute] int id, [FromForm] bool updateSlu
                 {
                     post.PubDate = DateTimeOffset.Now;
                 }
+                if (post.Slug == "")

```

Comment:
Can you use `hasSlug` in this if-statement? Or maybe `string.IsNullOrEmpty(post.Slug)`?

Reply:
Yeah, stylistically, that would probably be better. I'll change it, thanks!